allennlp/models/model.py

```diff
@@ -333,6 +335,61 @@ def extend_embedder_vocab(self, extended_vocab: Vocabulary) -> None:
             if isinstance(module, (Embedding, TextFieldEmbedder)):
                 module.extend_vocab(extended_vocab)
 
+    def inspection_dict(self,
```

Comment:
I would rather this not be a method on `Model` (it doesn't seem to access any private variables), why not just make it a helper function in `allennlp.nn.util`?

Reply:
also, I don't like the name "inspection_dict", that doesn't tell me anything about what it does. I would write two functions and call one `inspect_modules` and one `inspect_parameters`. But why have separate methods for those? Why not just inspect everything at once?